lib/compiler-singlepass/src/codegen.rs

```diff
@@ -3181,6 +3251,9 @@ impl<'a, M: Machine> FuncGen<'a, M> {
                     [Location::Imm32(memory_index.index() as u32), dst, val, len]
                         .iter()
                         .cloned(),
+                    [WpType::I32, WpType::I64, WpType::I64, WpType::I64]
+                        .iter()
+                        .cloned(),
```

Comment:
Isn't it better to do `into_iter()` ?

Reply:
Sure, I can use `into_iter()` instead.

Reply:
But I'll still need a `.cloned()` so I think it will not change much in the end.